src/styles/App.scss

```diff
@@ -11,7 +11,7 @@ h1 {
   text-align: center;
   font-size: 60px;
   padding: 20px;
-  text-shadow: 1px 1px 2px black, 0 0 25px white, 0 0 5px gold;
+  text-shadow: 1px 1px 2px black, 0 0 25px white, 0 0 5px $textColor;
```

Comment:
This might be my new favorite effect! Looks absolutely amazing! Thank you for adding this!